core-plugins/docs/File-batchsink.md

```diff
@@ -3,13 +3,16 @@
 
```

Comment:
Please provide a enduser use case as well.

Reply:
It wasn't in the original file, it's better that the use case  comes from you. I'll edit it language wise if needed

Reply:
so use case should be like this . 
apply any any source plugin(es,file,kafka, etc) -> file sink .

Reply:
This does not seem to be going with the first line in this document. Please suggest a complete use case

Reply:
Consider a scenario wherein you need to write a file to an HDFS in batch. For example, you may want to periodically dump any RDD data to HDFS in the formate of files(csv,tsv,json etc.) .